src/lib/dai.c

```diff
@@ -222,6 +225,8 @@ struct dai *dai_get(uint32_t type, uint32_t index, uint32_t flags)
 	dai_set_device_params(d);
 
 	if (dai_probe(d->dev)) {
+		tr_err(&dai_tr, "dai_get: failed to probe dai with index %d type %d",
+		       index, type);
```

Comment:
as an option, you could print the returned error code

Reply:
Lets add this incrementally, would be good to get weekend tests done with this log.